src/components/UI/Footer/Footer.js

```diff
@@ -0,0 +1,44 @@
+import facebookIcon from '../../../images/icons/facebook-brands.svg';
+import instagramIcon from '../../../images/icons/instagram-brands.svg';
+import pinterestIcon from '../../../images/icons/pinterest-brands.svg';
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="footer__section">
+        <div className="footer__section__title">ABOUT US</div>
+        <p className="footer__section__description">
+          Lorem ipsum dolor sit amet cons adipisicing elit sed do eiusm tempor
+          incididunt ut labor et dolore magna aliqua. Ut enim ad minim veniam,
+          quis nostrud.
+        </p>
+        <div className="footer__section__icons">
+          <img src={facebookIcon} />
+          <img src={instagramIcon} />
+          <img src={pinterestIcon} />
```

Comment:
Mostly we use something like this instead of <img /> with SVG files:
```
import { ReactComponent as Icon } from './icon.svg';

<Icon />
```

Reply:
nice way!